### PR TITLE
Add Spectrum BP-01-001, BP-01-002 ,BP-01-003, BP-02-001 tasks.

### PR DESCRIPTION
**Pull request by kernelpanic9**
- Add Spectrum BP-01-001, BP-01-003 tasks.
- Revert "Add Spectrum BP-01-001, BP-01-003 tasks."
- Add Spectrum BP-01-001, BP-01-002 ,BP-01-003 tasks.

### CODE REVIEW

File: testnet/tasks/BP-01-003/cYksJYkECwPw.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "uid": "cYksJYkECwPw"
```
Comment (CloverRL) [inline testnet/tasks/BP-01-003/cYksJYkECwPw.json:2]: re-01-003

File: testnet/tasks/BP-01-001/cYksJYkECwPw.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "uid": "cYksJYkECwPw",
+    "api_key_name": "testkey"
+}
```
Comment (0xcoco) [inline testnet/tasks/BP-01-001/cYksJYkECwPw.json:4]: reviewed 01-001

Comment (0xcoco) [inline testnet/tasks/BP-01-001/cYksJYkECwPw.json:4]: reviewed 01-002

Comment (0xcoco) [inline testnet/tasks/BP-01-001/cYksJYkECwPw.json:4]: reviewed 01-003

Comment (0xcoco) [inline testnet/tasks/BP-01-001/cYksJYkECwPw.json:4]: reviewed 02-002

**Comment by 0xcoco** [comment]
reviewed 02-001

**Pull request closed without merging**